text/0000-filesystem-integration.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: Filesystem Integration
+- Start Date: 2019-07-02
+- RFC PR:
+- Issue:
+
+# Summary
+[summary]: #summary
+
+Adds filesystem integration to qri datasets including a current working directory context for some qri commands.  The goal is to provide an experience that is more analogous to the git workflow, where working changes to files are sensed prior to committing a new version.
+
+# Motivation
+[motivation]: #motivation
+
+The motivation for this feature is to provide a more tangible connection between the qri versioning/publishing ecosystem and the file-based data that data practitioners are used to working with. Since all of the information in a qri dataset maps to files, we should just make those files exist automatically when a qri dataset is "added" to the user's local machine.  Likewise, we should watch for changes to those files, and ensure that they are all in a valid state before allowing the user to commit changes to a qri dataset.
+
+*This will significantly lower the barrier to entry for new users to qri, especially GUI users, as they can quickly pull down published datasets and begin working with them without having to understand how IPFS/QRI stores and maintains the data.*  
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The intent is to create a workflow more analogous with the git workflow.  When a new user wants the files in a remote git repo, they use `git clone {repoUrl}` and a new directory is created on their filesystem containing those files.  They can use _whatever tooling they want_ to make changes to the files, and commit them locally when they reach a critical point.
```

Comment:
I don't think the intent is to be more analogous with git, but rather to _use a common interface_: the filesystem. This is analogous to git, which is nice, but there are subtle differences we'll need to avail users to.

I want to keep us focused on taking inspiration from the good bits of existing version control systems. We should aim to do better than git's workflow when it comes to versioning data

Reply:
👍 Added the bits about using a "common interface"